tests/foreman/ui/test_activationkey.py

```diff
@@ -320,7 +320,7 @@ def test_negative_create_with_invalid_limit(self):
                         limit=limit,
                     )
                     self.assertIsNotNone(self.activationkey.wait_until_element(
-                        locators['ak.invalid_limit']))
+                        locators['hostcollection.invalid_limit']))
```

Comment:
It is really confusing to see hostcollection locator used in activation key test